packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -613,4 +632,12 @@ export {configureApp, isPackageOrBundle, duplicateKeys, parseCapsArray, generate
  * @property {Date?} lastModified
  * @property {boolean} immutable
  * @property {number?} maxAge
+ * @property {string?} etag
+ */
+
+/**
+ * @typedef RemoteAppData
```

Comment:
please add a description and descriptions of props

Reply:
np, added descriptions